### PR TITLE
Add text item with one click

### PR DESCRIPTION
**Pull request by alex-deaconu**
Added item buttons for playlist item
Buttons only show for system administrator role users (Rise)

Re-named some functions in playlist item factory

[stage-3]

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. 

Regarding the new buttons, they are only available for System Administrators, however I can remove before merging if you feel it may create confusion. 

Alternately, we can not merge this PR, and continue working on this branch until all functionality is complete. Thoughts?

Thanks!

**Comment by ezequielc** [comment]
@alex-deaconu I think we can hold merging and work on this branch all the functionality. It will cause less noise.
I will review the code as soon as I get a chance. Cheers

File: web/partials/editor/placeholder-playlist.html [hunk 1]
```diff
@@ -3,6 +3,42 @@
...
+    </a>
+    <a href="#" class="btn btn-default" ng-click="addItemShortcut('video')">
+      <image src="https://s3.amazonaws.com/ux-design/sprint_prototype/noun_14211_cc.svg"></image>
+      <span style="
+      font-size: 13px;
+      display: block;
+      text-align: center;
+      line-height: 15px;">Video</span></a></a>
+    <a href="#" class="btn btn-default" ng-click="playlistItemFactory.addTextWidget()">
+      <image src="https://s3.amazonaws.com/ux-design/sprint_prototype/noun_508199_cc.svg"></image>
```
Comment (ezequielc) [inline web/partials/editor/placeholder-playlist.html]: @alex-deaconu Should we move this to a definitive folder? Maybe @RV-Adrian can do that when implementing the other actions. Thanks

File: web/scripts/config/prod.js [hunk 2]
```diff
@@ -21,6 +21,7 @@
     .value('VIEWER_URL', 'http://preview.risevision.com')
     .value('ALERTS_WS_URL',
       'https://rvaserver2.appspot.com/alerts/cap')
+    .value('TEXT_WIDGET_ID', '32d460d1-a727-4765-a8e9-587f7915ab05')
```
Comment (ezequielc) [inline web/scripts/config/prod.js]: @alex-deaconu We have the IDs here as well: https://github.com/Rise-Vision/rise-vision-apps/blob/master/web/scripts/editor/services/svc-widget-renderer.js#L6
We might need a refactoring. Thoughts? Thanks

File: web/partials/editor/placeholder-playlist.html (hunk 1 above)
Comment (RV-Adrian) [inline web/partials/editor/placeholder-playlist.html]: @ezequielc The inline CSS here? Sure, I can move that over when ready. 👍

File: web/scripts/config/prod.js (hunk 2 above)
Comment (alex-deaconu) [inline web/scripts/config/prod.js]: @ezequielc good catch. Will revisit, have to figure out a way to merge the two. Thanks!

File: web/partials/editor/placeholder-playlist.html (hunk 1 above)
Comment (ezequielc) [inline web/partials/editor/placeholder-playlist.html]: @RV-Adrian Actually, I was talking about the location of the svg file, that folder is from design sprint, and might be deleted accidentally.

I missed the CSS, but that should be moved too :) 

Thanks

Comment (alex-deaconu) [inline web/partials/editor/placeholder-playlist.html]: @RV-Adrian I believe @ezequielc was referring to the SVG icons. My suggestion is we put them here: https://github.com/Rise-Vision/common-svg/blob/master/src/icons.js. 

Also, those icons are quite old, are there newer versions I should be using? 

Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu Code looks good. Just notes on refactoring Widget IDs config and SVG files location. Cheers

**Comment by alex-deaconu** [comment]
@ezequielc thanks for the review! Will address those changes in future PRs for this branch.

File: web/scripts/config/prod.js (hunk 2 above)
Comment (RV-Adrian) [inline web/scripts/config/prod.js]: @alex-deaconu @ezequielc ah! Yes, that sounds like an ideal spot.

**Comment by ezequielc** [comment]
@RV-Adrian Another thing I realized here, currently, after adding a widget we show the Playlist Item Settings modal. Just to confirm, we are not going to show it anymore, right? Thanks

**Comment by RV-Adrian** [comment]
@ezequielc Correct, we're aiming for discoverability and not to push unexpected visual elements to the users without context. Thx!

**Comment by ezequielc** [comment]
@RV-Adrian Great, thanks!

**Pull request merged**